antarest/study/storage/rawstudy/model/filesystem/config/links.py

```diff
@@ -66,6 +66,42 @@ class TransmissionCapacity(EnumIgnoreCase):
     ENABLED = "enabled"
 
 
+class LinkStyle(EnumIgnoreCase):
+    """
+    Enum representing the style of a link in a network visualization.
+
+    Attributes:
+        DOT: Represents a dotted line style.
+        PLAIN: Represents a solid line style.
+        DASH: Represents a dashed line style.
+        DOT_DASH: Represents a line style with alternating dots and dashes.
+    """
+
+    DOT = "dot"
+    PLAIN = "plain"
+    DASH = "dash"
+    DOT_DASH = "dotdash"
+
+
+class FilterOption(EnumIgnoreCase):
```

Comment:
You should probably use the class `FilteringOptions` inside file /variantstudy/model/command/common.py